### PR TITLE
Switch material properties in PFPressure to AD

### PR DESCRIPTION
**Pull request by andrembcosta**

### CODE REVIEW

File: benchmarks/sneddon/mechanical_fracture_uo_tet.i [hunk 1]
```diff
@@ -0,0 +1,224 @@
+E = 2.1e5
```
Comment (hugary1995) [inline benchmarks/sneddon/mechanical_fracture_uo_tet.i:1]: what's the deal with this input file? maybe remove it?

File: benchmarks/sneddon/mechanical_fracture_uo.i [hunk 2]
```diff
@@ -0,0 +1,229 @@
...
+  [./pff_pressure]
+    type = ADPFFPressure
+    variable = 'd'
+    pressure_uo = 'pressure_uo'
+    displacements = 'disp_x disp_y'
+  [../]
+[]
+
+[BCs]
+  # [./ydisp]
```
Comment (hugary1995) [inline benchmarks/sneddon/mechanical_fracture_uo.i:112]: remove comments

File: benchmarks/sneddon/mechanical_fracture_uo_tet.i (hunk 1 above)
Comment (andrembcosta) [reply benchmarks/sneddon/mechanical_fracture_uo_tet.i:1]: I don't know. I just copied the mode1 example and changed the other file. I think we can delete this one

File: benchmarks/sneddon/mechanical_fracture_uo.i (hunk 2 above)
Comment (andrembcosta) [reply benchmarks/sneddon/mechanical_fracture_uo.i:112]: ok

I just need to remove and commit again?

File: benchmarks/sneddon/mechanical_fracture_uo.i [hunk 3]
```diff
@@ -0,0 +1,229 @@
+E = 1
+nu = 0.2
+Gc = 1
+l = 0.1
+psic = 0
+k = 1e-6
+dc = 0
```
Comment (hugary1995) [inline benchmarks/sneddon/mechanical_fracture_uo.i:7]: I think for sneddon you need dc >= 1

File: benchmarks/sneddon/mechanical_fracture_uo.i [hunk 4]
```diff
@@ -0,0 +1,229 @@
...
+  [./disp_y]
+  [../]
+  [./d]
+  [../]
+[]
+
+[AuxVariables]
+  [./bounds_dummy]
+  [../]
+  [./fy]
```
Comment (hugary1995) [inline benchmarks/sneddon/mechanical_fracture_uo.i:30]: remove fy

File: benchmarks/sneddon/mechanical_fracture_uo.i [hunk 5]
```diff
@@ -0,0 +1,229 @@
...
+    variable = 'disp_x'
+    component = 0
+    displacements = 'disp_x disp_y'
+  [../]
+  [./solid_y]
+    type = ADStressDivergenceTensors
+    variable = 'disp_y'
+    component = 1
+    displacements = 'disp_x disp_y'
+    save_in = 'fy'
```
Comment (hugary1995) [inline benchmarks/sneddon/mechanical_fracture_uo.i:73]: remove save_in

File: benchmarks/sneddon/mechanical_fracture_uo.i [hunk 6]
```diff
@@ -0,0 +1,229 @@
...
+    local_dissipation_norm = 2
+  [../]
+  [./degradation]
+    type = QuadraticDegradation
+    d = 'd'
+    residual_degradation = ${k}
+  [../]
+[]
+
+[Postprocessors]
```
Comment (hugary1995) [inline benchmarks/sneddon/mechanical_fracture_uo.i:187]: remove postprocessor

File: benchmarks/sneddon/mechanical_fracture_uo.i [hunk 7]
```diff
@@ -0,0 +1,229 @@
...
+  compute_scaling_once = false
+
+  fp_max_its = 100
+  fp_tol = 1e-06
+  accept_on_max_fp_iteration = true
+[]
+
+[Outputs]
+  print_linear_residuals = false
+  [./csv]
```
Comment (hugary1995) [inline benchmarks/sneddon/mechanical_fracture_uo.i:216]: remove csv output

File: benchmarks/sneddon/mechanical_fracture_uo.i (hunk 3 above)
Comment (andrembcosta) [reply benchmarks/sneddon/mechanical_fracture_uo.i:7]: Ohhhhh. That's right. That might be why the stress decomposition is not working

**Pull request merged**